coreferee/lang/fr/language_specific_rules.py

```diff
@@ -330,7 +330,7 @@ def is_potential_anaphor(self, token: Token) -> bool:
         # impersonal constructions
         if (
             token.dep_ in {"expl:comp", "expl:pass", "expl:subj"}
-            and token.lemma_ not in {"y", "en"}
+            and token.lemma_ not in {"en"}
             and not self.has_morph(token, "Reflex", "Yes")
         ):
             return False
```

Comment:
Here an example (with `_lg` and `_md` models) is `J'y habite`. `y` gets `dep_ == "expl:comp"`.

Reply:
Yes, I had removed "y" from this because "y"'s dep gets way less frequently misclassified than "en"'s dep.

I think missing some of the actual "y" anaphors is a worthy sacritice as to be able to discard the actual  "y" expletives.